Simplified/NYPLConfiguration.m

```diff
@@ -22,12 +22,12 @@ + (void)initialize
   }
   
 #if defined(FEATURE_DRM_CONNECTOR)
-  [[NYPLADEPT sharedInstance] setLogCallback:^(__unused NSString *logLevel,
-                                               NSString *const exceptionName,
-                                               __unused NSDictionary *data,
-                                               NSString *const message) {
-    NSLog(@"ADEPT: %@: %@", exceptionName, message);
-  }];
+//  [[NYPLADEPT sharedInstance] setLogCallback:^(__unused NSString *logLevel,
```

Comment:
For some reason this callback was creating duplicate adept messages in the debugger..